scripts/combine/setupCombine.py

```diff
@@ -395,9 +395,15 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
             passToFakes=passSystToFakes,
         )
         if args.theoryAgnostic:
+            import numpy as np
             # open file with theory bands
             with h5py.File(f"{common.data_dir}/angularCoefficients/theoryband_variations.hdf5", "r") as ff:
                 scale_hists = narf.ioutils.pickle_load_h5py(ff["theorybands"])
+                for tt in scale_hists:
```

Comment:
Although it's fine to have it here, I am wondering whether it would be better to have a function imported from elsewhere and called here to apply this operation. Perhaps the function might also take care of loading the histograms given the file name

Reply:
Agreed with this suggestion